17 февр. 2022 18:31:44,225 system - 127.0.0.1 - 127.0.0.1 ERROR SQLLogger - Failed to interrupt process Thread[pool-9-Training.persons-pausable-daemon-1,5,RMI Runtime]: no private connection found
